### PR TITLE
3 /api

### PR DESCRIPTION
**Pull request by Cody2612**
created /api on the app.js
created test suit for the /api endpoint

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -12,20 +13,32 @@ afterAll(() => {
...
 
-  describe("/api/anything", () => {
+describe("/api", ()=>{
+    test("Get: 200 - responds with all the endpoints", ()=>{
+        return request(app)
+        .get("/api")
+        .expect(200)
+        .then(({ body }) => {
+            expect(typeof body.endpoints).toBe("object");
+            expect(body.endpoints).toEqual(endpoints);
```
Comment (JoWatson2011) [inline __tests__/app.test.js:23]: 🌟 🥳 
I think the test on line 22 is not entirely necessary, as this test will fail if body.endpoints is not an object. If you prefer to have it, that's fine too 😄

File: __tests__/app.test.js [hunk 2]
```diff
@@ -12,20 +13,32 @@ afterAll(() => {
...
+        });
+    })
+})
+
+describe("/api/anything", () => {
```
Comment (JoWatson2011) [inline __tests__/app.test.js:28]: Let's change this describe block description to some thing more descriptive - perhaps something like "General error handling" ?

File: app.js [hunk 3]
```diff
@@ -1,6 +1,11 @@
 const express = require("express");
 const { getTopics } = require("./controllers/topics.controllers");
 const app = express();
+const endpoints = require("./endpoints.json");
+
+app.get("/api", (request, response) =>{
+    response.status(200).send({endpoints});
+})
```
Comment (JoWatson2011) [inline app.js:8]: 🌟 
Really nice job getting your head around this! I hope taking a break helped 👍 If you want to refactor after this week is done, consider moving this callback function into it's own controller function file, to make it more consistent with your other endpoints.

**Review by JoWatson2011 — COMMENTED** [review]
Nice work Bogdan 👏 👏

File: __tests__/app.test.js (hunk 1 above)
Comment (Cody2612) [reply __tests__/app.test.js:23]: I will leave this one for now, I am having a question.
Should I create a new branch specifc to changes or can do this in my current branch and commit normally, having this change once i make the final push ?

File: app.js (hunk 3 above)
Comment (Cody2612) [reply app.js:8]: I wanted to do it with the controller initially but I said I want to finish the tasks...I guess speed does involve a loss of quality at my current level

**Pull request merged**